### PR TITLE
Adding docs for generating visual

### PR DESCRIPTION
**Pull request by akalburgi**
### Pull Request Checklist

- [ ] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/master/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [ ] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [ ] Add a line to the [release notes](https://github.com/digital-asset/daml/blob/master/docs/source/support/release-notes.rst), if appropriate

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

File: docs/source/experimental/warning.rst [hunk 1]
```diff
@@ -12,3 +12,4 @@ The tools described in this section are actively being designed and are subject
 
    Navigator Console </tools/navigator/console>
    Extractor </tools/extractor>
+   Visualization </tools/visual>
```
Comment (neil-da) [inline docs/source/experimental/warning.rst]: I don't believe in experimental tools - they are just tools you are saying are a bit crappy. Can we just put it as a normal tool.

File: docs/source/tools/visual.rst [hunk 2]
```diff
@@ -0,0 +1,32 @@
+.. Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
```
Comment (neil-da) [inline docs/source/tools/visual.rst]: Can we put an example output in the docs? ideally one generated from the quickstart, so we can tell users how to reproduce it.

File: docs/source/experimental/warning.rst (hunk 1 above)
Comment (bethaitman) [reply docs/source/experimental/warning.rst]: Experimental is only for stuff that we're not really happy to support properly. Everything else should just be in the normal tools section

File: docs/source/tools/visual.rst [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+.. SPDX-License-Identifier: Apache-2.0
+
+DAML Contract Visulization
+#########
```
Comment (bethaitman) [inline docs/source/tools/visual.rst]: This isn't gonna work, unfortunately - needs to be as long as the line above

File: docs/source/tools/visual.rst [hunk 4]
```diff
@@ -0,0 +1,32 @@
+.. Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+DAML Contract Visulization
```
Comment (bethaitman) [inline docs/source/tools/visual.rst]: Task-based headings are normally better for users - so something like "Visualizing DAML contracts"

(Plus you don't need to use Title Case, just capitalise like you would in a sentence)

File: docs/source/tools/visual.rst [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+#########
+
+Introduction
```
Comment (bethaitman) [inline docs/source/tools/visual.rst]: This is such a short page, I don't think it needs this heading

File: docs/source/tools/visual.rst [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+************
+
+You can generate visual graphs for DAML contracts visual command via command line tool. This program generates dot file which in turn can fed in to graphviz engine to produce an image.
```
Comment (bethaitman) [inline docs/source/tools/visual.rst]: This is a bit ambiguous as to what the user actively needs to do. And I think you can  merge this with the instructions below. I'd suggest something like: 
 
```suggestion
You can generate visual graphs for the contracts in your DAML project. To do this:

 1. Install `Graphviz <http://www.graphviz.org/download/>`_.
 2. Generate a DAR from your project by running ``daml build``.
 3. Generate a `dot file <https://en.wikipedia.org/wiki/DOT_(graph_description_language)>`_ from that DAR by running ``daml damlc visual <path_to_project>/dist/<project_name.dar> --dot <project_name>.dot``
 4. Generate the visual graph with Graphviz by running ``dot -Tpng <project_name>.dot > <project_name>.dot.png``
```

File: docs/source/tools/visual.rst (hunk 3 above)
Comment (neil-da) [reply docs/source/tools/visual.rst]: 😢

**Comment by bethaitman** [comment]
Oh, and an entry in the release notes would be great, too!

File: docs/source/tools/visual.rst (hunk 4 above)
Comment (hurryabit) [reply docs/source/tools/visual.rst]: There's also a typo in "Visulization".

File: docs/source/tools/visual.rst [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+
+Trying it out
+*************
```
Comment (hurryabit) [inline docs/source/tools/visual.rst]: Do we need an empty line below this heading for the rendering to work? Same below.

File: docs/source/experimental/warning.rst (hunk 1 above)
Comment (akalburgi) [reply docs/source/experimental/warning.rst]: Cool will remove it from this warning page

File: docs/source/tools/visual.rst (hunk 4 above)
Comment (akalburgi) [reply docs/source/tools/visual.rst]: :(

File: docs/source/tools/visual.rst (hunk 6 above)
Comment (akalburgi) [reply docs/source/tools/visual.rst]: Cool adapted the changes. This looks a lot easier.

File: docs/source/tools/visual.rst [hunk 8]
```diff
@@ -0,0 +1,11 @@
+.. Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+Visualizing DAML Contract
```
Comment (bethaitman) [inline docs/source/tools/visual.rst]:
```suggestion
Visualizing DAML contracts
```

Comment (bethaitman) [inline docs/source/tools/visual.rst]: and you still need the underline otherwise it's not a title!

Comment (akalburgi) [reply docs/source/tools/visual.rst]: I am assuming ### is used for title correct me if I am wrong, please

File: docs/source/tools/visual.rst (hunk 7 above)
Comment (akalburgi) [reply docs/source/tools/visual.rst]: Nope, not needed it. Removed

**Comment by akalburgi** [comment]
> I still think this would benefit from an example of what the output looks like, but this is now reasonably complete

I agree. Will add one in the next iteration which should be very soon.

File: docs/source/tools/visual.rst (hunk 2 above)
Comment (akalburgi) [reply docs/source/tools/visual.rst]: Will add it very soon. In a different PR

**Pull request merged**